sunpy/time/time.py

```diff
@@ -195,6 +195,20 @@ def convert_time_astropy(time_string, **kwargs):
     return time_string
 
 
+@convert_time.register(list)
+def convert_time_list(time_list, format=None, **kwargs):
+    item = time_list[0]
+    # If we have a list of strings, need to get the correct format from our
+    # list of custom formats
+    if isinstance(item, str):
+        if not format:
+            format = _get_time_fmt(item)
+        return Time.strptime(time_list, format, **kwargs)
```

Comment:
Test failure is real.  The `format` that can get passed in is a `Time` format (e.g., `'isot'`), but you're passing that on as if it were a string format (e.g., `'%Y-%m-%d'`).

Reply:
Thanks - I think this is fixed now. I assumed if a user passed a custom format they just wanted to use `Time`, so avoided doing anything else in that case.